### PR TITLE
Adding sysvar epochSchedule, rent, slotHashes accounts 

### LINKED ISSUE #1: Account components: Sysvars
Add support for _some_ sysvars as account components. Note I've intentionally
left a few out, since we don't need to support everything in the demo.

These can simply copy the existing patterns laid down for the other account
components.

- [x] Clock
- [x] Epoch Schedule
- [x] Rent
- [x] Slot Hashes

### PR DESCRIPTION
**Pull request by nasjuice**
This PR adds Sysvar epochSchedule rent slotHashes

Continuing work on https://github.com/buffalojoec/graphql-wallet/issues/1

### CODE REVIEW

**Comment by nasjuice** [comment]
The SysvarEpochRewards1111111111111111111111111 doesn't seem to show on my local rpc

File: src/components/accounts/SysvarEpochScheduleAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+/**
+ * Component GraphQL data.
+ */
+type Data = {
+    firstNormalEpoch: Epoch;
+    firstNormalSlot: Slot;
+    leaderScheduleSlotOffset: BigInt;
+    slotsPerEpoch: BigInt;
+    warmup: Boolean;
```
Comment (buffalojoec) [inline src/components/accounts/SysvarEpochScheduleAccount.tsx]: Can we use primitives here? `bigint` and `boolean`?

File: src/components/accounts/SysvarEpochScheduleAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    }, []);
+
+    if (data) {
+        return (
+            <div>
+                <p>First Normal Epoch: {data.firstNormalEpoch.toString()}</p>
+                <p>First Normal Slot: {data.firstNormalSlot.toString()}</p>
+                <p>Leader Schedule Slot Offset {data.leaderScheduleSlotOffset.toString()}</p>
+                <p>Slots Per Epoch: {data.slotsPerEpoch.toString()}</p>
+                <p>Warmup: {data.warmup.toString()}</p>
```
Comment (buffalojoec) [inline src/components/accounts/SysvarEpochScheduleAccount.tsx]: Shouldn't need `toString()` on primitive `boolean`.

File: src/components/accounts/SysvarSlotHashesAccount.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+/**
+ * Component GraphQL data.
+ */
+type Data = {
+    entries: SlotHashEntry[];
+};
+
+type SlotHashEntry = {
+    hash: String;
```
Comment (buffalojoec) [inline src/components/accounts/SysvarSlotHashesAccount.tsx]: Primitive here as well `string`.

**Review by buffalojoec — COMMENTED** [review]
Looks good, just some tiny suggestions.

**Comment by nasjuice** [comment]
Thank you its fixed

**Comment by buffalojoec** [comment]
> The SysvarEpochRewards1111111111111111111111111 doesn't seem to show on my local rpc

It won't always be there unfortunately. Let's actually skip that one. I'll remove it from the issue.

**Review by buffalojoec — APPROVED** [review]
Nice! Thanks!

**Pull request merged**